docs/testing/reference.md

```diff
@@ -102,15 +102,26 @@ commands:
 collectors:
 - type: pod
   pod: nginx
+resourceRefs:
+  - apiVersion: apps/v1
+    kind: Deployment
+    namespace: kube-system
+    name: coredns
+    ref: coredns
+assertAll:
+  - celExpr: "coredns.spec.replicas >= 2"
```

Comment:
Let's keep the indentation consistent, and use a ref different from the name just to make it easier to see what is what from the example:
```suggestion
- apiVersion: apps/v1
  kind: Deployment
  namespace: kube-system
  name: coredns
  ref: coredns_deployment
assertAll:
- celExpr: "coredns_deployment.spec.replicas >= 2"
```